docs/tutorials/install.rst

```diff
@@ -210,6 +210,32 @@ Assuming `brew <http://brew.sh/>`_ is installed:
     pip install virtualenv
 
 
+
+Microsoft Windows
```

Comment:
Can we put this in another pull-request and keep this one about pip update?

Reply:
Of course, you got it. I just was about to remove it for know because it should be edited anyway. I'll make an update in another request :)